[2022-07-23 05:27:57,070] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: Data_Quality_Pipeline.upload_to_s3_2019-02 manual__2022-07-23T05:27:47.881604+00:00 [queued]>
[2022-07-23 05:27:57,148] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: Data_Quality_Pipeline.upload_to_s3_2019-02 manual__2022-07-23T05:27:47.881604+00:00 [queued]>
[2022-07-23 05:27:57,148] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-07-23 05:27:57,148] {taskinstance.py:1377} INFO - Starting attempt 1 of 1
[2022-07-23 05:27:57,148] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-07-23 05:27:57,266] {taskinstance.py:1397} INFO - Executing <Task(LocalFilesystemToS3Operator): upload_to_s3_2019-02> on 2022-07-23 05:27:47.881604+00:00
[2022-07-23 05:27:57,309] {standard_task_runner.py:52} INFO - Started process 38773 to run task
[2022-07-23 05:27:57,346] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'Data_Quality_Pipeline', 'upload_to_s3_2019-02', 'manual__2022-07-23T05:27:47.881604+00:00', '--job-id', '36', '--raw', '--subdir', 'DAGS_FOLDER/localtos3.py', '--cfg-path', '/tmp/tmp2m649osy', '--error-file', '/tmp/tmpm775r7wq']
[2022-07-23 05:27:57,360] {standard_task_runner.py:80} INFO - Job 36: Subtask upload_to_s3_2019-02
[2022-07-23 05:27:57,750] {task_command.py:371} INFO - Running <TaskInstance: Data_Quality_Pipeline.upload_to_s3_2019-02 manual__2022-07-23T05:27:47.881604+00:00 [running]> on host d30608300872
[2022-07-23 05:27:58,148] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=Data_Quality_Pipeline
AIRFLOW_CTX_TASK_ID=upload_to_s3_2019-02
AIRFLOW_CTX_EXECUTION_DATE=2022-07-23T05:27:47.881604+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-07-23T05:27:47.881604+00:00
[2022-07-23 05:27:58,192] {base.py:68} INFO - Using connection ID 'aws-conn' for task execution.
[2022-07-23 05:27:58,193] {base_aws.py:210} INFO - Credentials retrieved from extra_config
[2022-07-23 05:27:59,562] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=Data_Quality_Pipeline, task_id=upload_to_s3_2019-02, execution_date=20220723T052747, start_date=20220723T052757, end_date=20220723T052759
[2022-07-23 05:27:59,663] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-07-23 05:27:59,797] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
